src/Internal.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace CoinParams\BitWasp;
+
+
+use CoinParams\Exceptions\ArrayKeyNotFound;
+
+class Internal
+{
+    /**
+     * @param $array array
+     * @param $key string
+     * @param bool|string $execption: type or message to throw an exeption with your message or false to return null.
+     * @return null|mixed
+     * @throws ArrayKeyNotFound
+     */
+    public static function aget($array , $key , $execption = false)
+    {
+        if (is_array($array) AND array_key_exists($key, $array) AND true == $array[$key]){
```

Comment:
1. exception param is spelled wrong.  typo?

2. the true check prevents it from ever returning a value of 0, as opposed to null.

This is different behavior from the @ operator on an array.    Example:

    $a = ['k' => 0];
    var_dump( @$a['k'] );
    var_dump( aget($a, 'k'));

Result:

    int(0)
    NULL

Reply:
Actually, its worth nothing to check if the value equals to true, let's just check if the array key exists and return its value whatever it is, what do you think?